<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>PersonDetailsForm</class>
 <widget class="QWidget" name="PersonDetailsForm">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>583</width>
    <height>524</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="font">
   <font>
    <pointsize>12</pointsize>
   </font>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QHBoxLayout" name="horizontalLayout_2">
   <item>
    <widget class="QGroupBox" name="groupBox">
     <property name="title">
      <string>Contact Information</string>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout">
      <item>
       <widget class="QGroupBox" name="addressGroupBox">
        <property name="title">
         <string>Address</string>
        </property>
        <layout class="QGridLayout" name="gridLayout_2">
         <item row="0" column="0">
          <widget class="QLabel" name="firstLabel">
           <property name="text">
            <string>First Name</string>
           </property>
           <property name="alignment">
            <set>Qt::AlignCenter</set>
           </property>
          </widget>
         </item>
         <item row="0" column="2">
          <widget class="QLabel" name="lastLabel">
           <property name="text">
            <string>Last Name</string>
           </property>
           <property name="alignment">
            <set>Qt::AlignCenter</set>
           </property>
          </widget>
         </item>
         <item row="1" column="1">
          <widget class="QLineEdit" name="middleLineEdit"/>
         </item>
         <item row="1" column="0">
          <widget class="QLineEdit" name="firstLineEdit"/>
         </item>
         <item row="0" column="1">
          <widget class="QLabel" name="middleLabel">
           <property name="text">
            <string>Middle</string>
           </property>
           <property name="alignment">
            <set>Qt::AlignCenter</set>
           </property>
          </widget>
         </item>
         <item row="1" column="2">
          <widget class="QLineEdit" name="lastLineEdit"/>
         </item>
         <item row="4" column="0" colspan="3">
          <widget class="QWidget" name="widget" native="true">
           <layout class="QGridLayout" name="gridLayout_3">
            <item row="0" column="1">
             <widget class="QLineEdit" name="streetLineEdit"/>
            </item>
            <item row="1" column="0">
             <widget class="QLabel" name="cityLabel">
              <property name="text">
               <string>City</string>
              </property>
             </widget>
            </item>
            <item row="2" column="1">
             <widget class="QLineEdit" name="countyLineEdit"/>
            </item>
            <item row="1" column="1">
             <widget class="QLineEdit" name="cityLineEdit"/>
            </item>
            <item row="0" column="2">
             <widget class="QLabel" name="unitLabel">
              <property name="text">
               <string>Unit</string>
              </property>
             </widget>
            </item>
            <item row="0" column="3">
             <widget class="QLineEdit" name="unitLineEdit"/>
            </item>
            <item row="2" column="0">
             <widget class="QLabel" name="countyLabel">
              <property name="text">
               <string>County</string>
              </property>
             </widget>
            </item>
            <item row="0" column="0">
             <widget class="QLabel" name="streetLabel">
              <property name="text">
               <string>Street</string>
              </property>
             </widget>
            </item>
            <item row="1" column="2">
             <widget class="QLabel" name="statelabel">
              <property name="text">
               <string>State</string>
              </property>
             </widget>
            </item>
            <item row="1" column="3">
             <widget class="QLineEdit" name="stateLineEdit"/>
            </item>
            <item row="2" column="2">
             <widget class="QLabel" name="zipLabel">
              <property name="text">
               <string>Zip</string>
              </property>
             </widget>
            </item>
            <item row="2" column="3">
             <widget class="QLineEdit" name="zipLineEdit"/>
            </item>
           </layout>
          </widget>
         </item>
        </layout>
       </widget>
      </item>
      <item>
       <widget class="QGroupBox" name="phoneGroupBox">
        <property name="title">
         <string>Phone and Email</string>
        </property>
        <layout class="QGridLayout" name="gridLayout_4">
         <item row="0" column="4">
          <widget class="QLineEdit" name="primaryExtLineEdit"/>
         </item>
         <item row="1" column="1">
          <widget class="QLineEdit" name="secondaryLineEdit"/>
         </item>
         <item row="1" column="0">
          <widget class="QLabel" name="homeLabel">
           <property name="text">
            <string>Secondary</string>
           </property>
          </widget>
         </item>
         <item row="0" column="0">
          <widget class="QLabel" name="workLabel">
           <property name="text">
            <string>Primary</string>
           </property>
          </widget>
         </item>
         <item row="1" column="4">
          <widget class="QLineEdit" name="secondaryExtLineEdit"/>
         </item>
         <item row="1" column="3">
          <widget class="QLabel" name="homeExtLabel">
           <property name="text">
            <string>Ext</string>
           </property>
          </widget>
         </item>
         <item row="0" column="1">
          <widget class="QLineEdit" name="primaryLineEdit"/>
         </item>
         <item row="0" column="3">
          <widget class="QLabel" name="workExtLabel">
           <property name="text">
            <string>Ext</string>
           </property>
          </widget>
         </item>
         <item row="2" column="0">
          <widget class="QLabel" name="emailLabel">
           <property name="text">
            <string>Email</string>
           </property>
          </widget>
         </item>
         <item row="2" column="1" colspan="3">
          <widget class="QLineEdit" name="emailLineEdit"/>
         </item>
        </layout>
       </widget>
      </item>
      <item>
       <widget class="QFrame" name="miscFrame">
        <property name="frameShape">
         <enum>QFrame::StyledPanel</enum>
        </property>
        <property name="frameShadow">
         <enum>QFrame::Raised</enum>
        </property>
        <layout class="QGridLayout" name="gridLayout_6">
         <item row="0" column="2">
          <widget class="QLineEdit" name="attorneyLineEdit"/>
         </item>
         <item row="0" column="1">
          <widget class="QLabel" name="attorneyLabel">
           <property name="text">
            <string>Attorney</string>
           </property>
          </widget>
         </item>
         <item row="0" column="0">
          <layout class="QFormLayout" name="formLayout">
           <item row="0" column="0">
            <widget class="QLabel" name="numInHomeLabel">
             <property name="text">
              <string>Adults In Home</string>
             </property>
            </widget>
           </item>
           <item row="0" column="1">
            <widget class="QLineEdit" name="numInHomeLineEdit"/>
           </item>
           <item row="1" column="0">
            <widget class="QLabel" name="label">
             <property name="text">
              <string>Chlildren In Home</string>
             </property>
            </widget>
           </item>
           <item row="1" column="1">
            <widget class="QLineEdit" name="numChildrenNHomeLineEdit"/>
           </item>
          </layout>
         </item>
        </layout>
       </widget>
      </item>
      <item>
       <widget class="QWidget" name="buttonContainer" native="true">
        <layout class="QHBoxLayout" name="horizontalLayout">
         <item>
          <widget class="QPushButton" name="deleteButton">
           <property name="text">
            <string>Delete</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="editButton">
           <property name="text">
            <string>Edit</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="saveButton">
           <property name="text">
            <string>Save</string>
           </property>
          </widget>
         </item>
        </layout>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
  </layout>
 </widget>
 <tabstops>
  <tabstop>firstLineEdit</tabstop>
  <tabstop>middleLineEdit</tabstop>
  <tabstop>lastLineEdit</tabstop>
  <tabstop>streetLineEdit</tabstop>
  <tabstop>unitLineEdit</tabstop>
  <tabstop>cityLineEdit</tabstop>
  <tabstop>countyLineEdit</tabstop>
  <tabstop>primaryLineEdit</tabstop>
  <tabstop>primaryExtLineEdit</tabstop>
  <tabstop>secondaryLineEdit</tabstop>
  <tabstop>secondaryExtLineEdit</tabstop>
  <tabstop>attorneyLineEdit</tabstop>
  <tabstop>saveButton</tabstop>
  <tabstop>deleteButton</tabstop>
  <tabstop>editButton</tabstop>
 </tabstops>
 <resources/>
 <connections/>
</ui>
